03-app-instrumentation.md

```diff
@@ -31,7 +56,16 @@ to add tracing & metrics. A few differences in your implementation:
 Give it a try yourself, if you are unsure how to accomplish this, you can peek
 into the [instrument.js](./app/frontend/instrument.js) file.
 
-Run that application either locally or by rebuilding the underlying container.
+To see if spans are emitted to the collector, call the frontend service via your
+browser or curl:
+
+```
+$ curl localhost:4000/
```

Comment:
The other commands in the tutorial don't start with  `$` (it's easier to copy paste)